pkg/cmd/release/create/create.go

```diff
@@ -516,12 +519,32 @@ func createRun(opts *CreateOptions) error {
 }
 
 func gitTagInfo(client *git.Client, tagName string) (string, error) {
-	cmd, err := client.Command(context.Background(), "tag", "--list", tagName, "--format=%(contents:subject)%0a%0a%(contents:body)")
+	contentCmd, err := client.Command(context.Background(), "tag", "--list", tagName, "--format=%(contents)")
 	if err != nil {
 		return "", err
 	}
-	b, err := cmd.Output()
-	return string(b), err
```

Comment:
Yeah. Since it's the result of a command execution, it's possible to have non-nil value for both error and result. But obviously, when there's an error the result value is unreliable. Esepcially, when it's the stdout of a command, which, for example, can be an error message.

Interestingly, the [caller](https://github.com/babakks/cli/blob/6dc729792d8942f2e53a2e9cd3c629e506b422a2/pkg/cmd/release/create/create.go#L261) of this function, ignores the error which makes it even more risky.

```go
tagDescription, _ = gitTagInfo(opts.GitClient, opts.TagName)
```

I think we should:
- Check the error at the caller side and print a proper error message when the tag annotation cannot be extracted.
- In `gitTagInfo`, return `"", err` when there's an error (as it's already done in this PR).

What do you think?